CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.0.6
```

Comment:
Maybe 1.1.0?

Reply:
+1 for 1.1.0, it'd be consistent with RN